index.html

```diff
@@ -65,6 +65,217 @@ <h3><a> Let's connect </a> </h3>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
+            <div class="view-project-2">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="portfolio-3">
+        <div class="project-3">
+            <div class="tonic-img-3">
+                <img src="Image/SnapshootPortfolio3.png" alt="error">
+            </div>
+            <div class="project-name-3">
+                <h1>Tonic</h1>
+            </div>
+            <div class="about-project-3">
+                <ul>
+                    <li class="company-3">CANOPY</li>
+                    <li class="position-3">Back End Dev</li>
+                    <li class="year-3">2015</li>
+                </ul>
+            </div>
+            <div class="role-project-3">
+                <p>
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-3">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
+            <div class="view-project-3">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="portfolio-4">
+        <div class="project-4">
+            <div class="tonic-img-4">
+                <img src="Image/SnapshootPortfolio4.png" alt="error">
+            </div>
+            <div class="project-name-4">
+                <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="about-project-4">
+                <ul>
+                    <li class="company-4">CANOPY</li>
+                    <li class="position-4">Back End Dev</li>
+                    <li class="year-4">2015</li>
+                </ul>
+            </div>
+            <div class="role-project-4">
+                <p>
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-4">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
+            <div class="view-project-4">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-me-page">
+
+            <div class="about-me-title">
+                <h1> About <br> Myself</h1>
+            </div>
+            <br>
+            <br>
```

Comment:
- [ ] Kindly delete all `<br>` tags, it is not semantic, use margin, padding to align content 👍🏿